packages/picasso/src/InputLimitAdornment/styles.ts

```diff
@@ -3,15 +3,13 @@ import { Theme, createStyles } from '@material-ui/core/styles'
 export default ({ palette }: Theme) =>
   createStyles({
     limiterLabel: {
-      color: palette.grey.main2,
       fontSize: '0.625rem',
-      lineHeight: '1rem',
+      lineHeight: 1,
+      marginTop: '0.25rem',
+      color: palette.grey.main2,
+      marginRight: '0.125rem',
```

Comment:
```suggestion
      marginRight: rem('2px'),
```
just to improve readability

Reply:
Do we have some guidelines on when to use this? Does it count only for really small values or should all the rems be written like this @TomasSlama ?

Reply:
I don't think we have a guideline, but it should be used to lower cognitive load. so for example, 1, 0.5, 0.25 rem is pretty clear, but otherwise, you need to think about it.